src/sdk/model/user/user.ts

```diff
@@ -327,4 +328,13 @@ export class User {
     return Iland.getHttp().delete(`/users/${this.username}/sessions`);
   }
 
+  /**
+   * Updates the user's profile picture.
+   * @param {number} pictureBytes
```

Comment:
The @param is not good. It should be something like : @param {JpegImage} picture